asclepius/asclepius/enrich.py

```diff
@@ -1,8 +1,40 @@
-def enrich(ddis):
-    def _enrich(ddi):
-        result = {'harmful': bool(hash(ddi['uri']) % 2),
-                  'spacing': abs(hash(ddi['uri']))%14 + 1}
-        result.update(ddi)
-        return result
+from hashlib import sha1
+import csv
+import sys
+import dinto
+import clize
+from clize.parameters import one_of
+
+def _sha_int(string):
+    return int(sha1(string.encode()).hexdigest()[-8:], 16)
```

Comment:
Ah, wouldn't work; Python only supports parsing to 2 ≤ base ≤ 26 numbers. I don't think the sha stuff is much of a bottleneck; it's dwarfed by the time to query Chiron in any case.